src/plugins/dashboard/public/dashboard_container/component/grid/_dashboard_grid.scss

```diff
@@ -36,10 +36,13 @@
 }
 
 /**
- * When a single panel is expanded, all the other panels are hidden in the grid.
+ * When a single panel is expanded, all the other panels moved offscreen.
+ * Shifting the rendered panels offscreen prevents panel reloading on minimize
  */
 .dshDashboardGrid__item--hidden {
-  display: none;
+  position: absolute; 
+  top: -9999px; 
+  left: -9999px;
```

Comment:
Shifting the position of all the hidden panels offscreen actually prevents a reload of the panels on minimize. When we toggle `display: none;`, you can see every panel loading when you minimize, but that additional loading goes away with this change.

Reply:
Hmmm, I'm not seeing that - took the following video with a throttled network so that the loading would be clear:


https://user-images.githubusercontent.com/8698078/234048922-5d78ef22-b4e2-4385-9f00-8e8ffcfbc2b2.mov

Reply:
I think I described the loading wrong. It doesn't trigger the panels to reload data wise with the whole loading bar visible. It's more of like a quick flash when redrawing the panels that goes away.

#### `display: none`

https://user-images.githubusercontent.com/1697105/234057570-22c5678e-aca9-46a5-966e-6c6cc62b28e5.mp4

#### Shifting the panels offscreen

https://user-images.githubusercontent.com/1697105/234057763-15d46d2b-3fea-422e-bd1b-aef1f1ff3932.mp4

Reply:
Oooh yes, seeing them side-by-side like that is really helpful thank you - that flicker is pretty easy to miss but you're totally right 💃